CMakeLists.txt

```diff
@@ -96,6 +96,11 @@ target_link_libraries(mirco
   ${MPI_LIBRARIES}
   )
 
+file(COPY ${CMAKE_SOURCE_DIR}/src/
+  DESTINATION include
+  FILES_MATCHING PATTERN "*.h*"
+)
```

Comment:
I did try that. I am getting the following error:
![image](https://user-images.githubusercontent.com/96046395/196200940-4d54c087-c10d-4948-a440-86a1354c19cf.png)
Also, I can't find the header files anywhere in the build directory.

Reply:
`CMakeLists.txt` does not yet define how to install MIRCO.

Some modifications to `CMakeLists.txt` are required. However, I don't know the exact syntax from the top of my head, since I don't do this everyday... actually, I probably have never set this up myself so far...

Reply:
I tried that, but this still does not automatically copy the header files to the `include/` directory.

Reply:
I tried the following:
set(CMAKE_INSTALL_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/newfolder")
INSTALL(TARGETS mirco
  LIBRARY DESTINATION lib
  PUBLIC_HEADER DESTINATION include
)

The `newfolder` is getting created inside the build directory with `bin`, `lib`, and `include`, but it still does not copy the header files.